src/OctoshiftCLI.Tests/gei/Commands/MigrateRepoCommandTests.cs

```diff
@@ -114,6 +116,69 @@ public async Task Happy_Path_Without_Wait()
+            var githubRepoUrl = $"https://github.com/{SOURCE_ORG}/{SOURCE_REPO}";
+            var migrationId = Guid.NewGuid().ToString();
+            var githubRepos = new List<string> { TARGET_REPO };
+
+            var mockGithub = new Mock<GithubApi>(null, null, null);
+            mockGithub.Setup(x => x.GetRepos(TARGET_ORG).Result).Returns(githubRepos);
+            mockGithub.Setup(x => x.StartMigration(migrationSourceId, githubRepoUrl, githubOrgId, TARGET_REPO, sourceGithubPat, targetGithubPat, "", "", false).Result).Returns(migrationId);
+
+            var environmentVariableProviderMock = new Mock<EnvironmentVariableProvider>(null);
+            environmentVariableProviderMock.Setup(m => m.SourceGithubPersonalAccessToken()).Returns(sourceGithubPat);
+            environmentVariableProviderMock.Setup(m => m.TargetGithubPersonalAccessToken()).Returns(targetGithubPat);
+
+            var mockGithubApiFactory = new Mock<ITargetGithubApiFactory>();
+            mockGithubApiFactory.Setup(m => m.Create(It.IsAny<string>(), It.IsAny<string>())).Returns(mockGithub.Object);
+
+            var actualLogOutput = new List<string>();
+            var mockLogger = new Mock<OctoLogger>();
+            mockLogger.Setup(m => m.LogInformation(It.IsAny<string>())).Callback<string>(s => actualLogOutput.Add(s));
+            mockLogger.Setup(m => m.LogWarning(It.IsAny<string>())).Callback<string>(s => actualLogOutput.Add(s));
+
+            var expectedLogOutput = new List<string>()
+            {
+                "Migrating Repo...",
+                $"GITHUB SOURCE ORG: {SOURCE_ORG}",
+                $"SOURCE REPO: {SOURCE_REPO}",
+                $"GITHUB TARGET ORG: {TARGET_ORG}",
+                $"TARGET REPO: {TARGET_REPO}",
+                $"TARGET API URL: {TARGET_API_URL}",
+                $"The Org '{TARGET_ORG}' already contains a repository with the name '{TARGET_REPO}'. No operation will be performed"
+            };
+
+            // Act
+            var command = new MigrateRepoCommand(mockLogger.Object, null, mockGithubApiFactory.Object, environmentVariableProviderMock.Object, null);
+            var args = new MigrateRepoCommandArgs
+            {
+                GithubSourceOrg = SOURCE_ORG,
+                SourceRepo = SOURCE_REPO,
+                GithubTargetOrg = TARGET_ORG,
+                TargetRepo = TARGET_REPO,
+                TargetApiUrl = TARGET_API_URL,
+                Wait = false
+            };
+            await command.Invoke(args);
+
+            // Assert
+            mockGithub.Verify(m => m.GetRepos(TARGET_ORG));
+
+            mockLogger.Verify(m => m.LogInformation(It.IsAny<string>()), Times.Exactly(6));
+            mockLogger.Verify(m => m.LogWarning(It.IsAny<string>()), Times.Exactly(1));
+            actualLogOutput.Should().Equal(expectedLogOutput);
+
+            mockGithub.VerifyNoOtherCalls();
+            mockLogger.VerifyNoOtherCalls();
```

Comment:
Base on my previous comment, these asserts can also be simplified. We can get rid of `LogInformation` verification and the `mockLogger.VerifyNoOtherCalls()` lines.
Again all we need to test here is to make sure that we return from the method with a warning log message.